src/hera.cpp

```diff
@@ -50,6 +52,49 @@ struct hera_instance : evm_instance {
 
 namespace {
 
+#if HERA_EVM2WASM_JS
+// NOTE: assumes that pattern doesn't contain any formatting characters (e.g. %)
+string mktemp_string(string pattern) {
+  char tmp[PATH_MAX] = { 0 };
+  size_t len = snprintf(tmp, sizeof(tmp), "%s", pattern.c_str());
+  mktemp(tmp);
+  return string(tmp, len);
+}
+
+vector<uint8_t> evm2wasm(vector<uint8_t> const& input) {
+  string fileEVM = mktemp_string("/tmp/hera.evm2wasm.evm.XXXXXXXX");
+  string fileWASM = mktemp_string("/tmp/hera.evm2wasm.wasm.XXXXXXXX");
+
+  ofstream os;
+  os.open(fileEVM);
+  // print as a hex sting
+  os << hex;
```

Comment:
Actually `evm2wasm` is broken. In the file it is hex, on the CLI it is hex string. Since in pretty much every case it is represented in a hex string I'd just fix `evm2wasm`.

Reply:
Fixed https://github.com/ewasm/evm2wasm/pull/194